warehouse/models/staging/transit_database/stg_transit_database__gtfs_service_data.sql

```diff
@@ -0,0 +1,47 @@
+{{ config(materialized='table') }}
+
+WITH
+latest AS (
+    {{ get_latest_external_data(
+        external_table = source('airtable', 'california_transit__gtfs_service_data'),
+        order_by = 'dt DESC, time DESC'
+        ) }}
+),
+
+stg_transit_database__gtfs_service_data AS (
+    SELECT
+        gtfs_service_data_id AS key,
+        {{ trim_make_empty_string_null(column_name = "name") }},
+        unnested_services AS services,
```

Comment:
service_key and below gtfs_dataset_key and reference_static_gtfs_service_data_key